src/app/[locale]/page.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Metadata } from 'next';
+import { draftMode } from 'next/headers';
+import Link from 'next/link';
+import { notFound } from 'next/navigation';
+
+import { ArticleHero, ArticleTileGrid } from '@src/components/features/article';
+import { Container } from '@src/components/shared/container';
+import TranslationsProvider from '@src/components/shared/i18n/TranslationProvider';
+import initTranslations from '@src/i18n';
+import { locales } from '@src/i18n/config';
+import { PageBlogPostOrder } from '@src/lib/__generated/sdk';
+import { client, previewClient } from '@src/lib/client';
+
+interface LandingPageProps {
+  params: {
+    locale: string;
+  };
+}
+
+export async function generateMetadata({ params }: LandingPageProps): Promise<Metadata> {
+  const { isEnabled: preview } = draftMode();
+  const gqlClient = preview ? previewClient : client;
+  const landingPageData = await gqlClient.pageLanding({ locale: params.locale, preview });
+  const page = landingPageData.pageLandingCollection?.items[0];
+
+  // optionally access and extend (rather than replace) parent metadata
+  // const previousImages = (await parent).openGraph?.images || [];
```

Comment:
Is this something intended for the user to comment out if they want additional functionality?

Reply:
I'll delete it, thanks!